<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1825</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>391</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_datasets"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Concentration</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Notes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="plotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>1300</width>
      <height>740</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_toolbox"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>10</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="horizontalSlider_plot_seconds">
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_plot_seconds">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_dataspace_buttons">
     <item>
      <widget class="QPushButton" name="pushButton_dataspace_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dataspace_remove">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_dataspace_rename">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_dataspaces"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1825</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_data_from_CSV"/>
    <addaction name="actionImport_data_from_xlsx"/>
    <addaction name="actionImport_data_from_PSSESSION"/>
   </widget>
   <widget class="QMenu" name="menuee">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionee"/>
    <addaction name="actionAutomatic_Import"/>
    <addaction name="actionDebug_Info"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuee"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_data_from_CSV">
   <property name="text">
    <string>Import data from CSV</string>
   </property>
  </action>
  <action name="actionImport_data_from_xlsx">
   <property name="text">
    <string>Import data from XLSX</string>
   </property>
  </action>
  <action name="actionee">
   <property name="text">
    <string>Layout</string>
   </property>
  </action>
  <action name="actionAutomatic_Import">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic Import</string>
   </property>
  </action>
  <action name="actionImport_data_from_PSSESSION">
   <property name="text">
    <string>Import data from PSSESSION</string>
   </property>
  </action>
  <action name="actionDebug_Info">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug Info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
